### PR TITLE
API for Owner and Renter 

### PR DESCRIPTION
**Pull request by Efrr2022**

### CODE REVIEW

File: amplify/backend/function/OwnerAndRenterFunction/src/index.py [hunk 1]
```diff
@@ -229,57 +228,103 @@ def save_user(request_body,userPath):
...
+def delete_user(id,userPath):
+  mycursor = mydb.cursor()
+  if userPath == '/owner':
+    sql = f"select * from tblOwner where ownerId={id}"
+    mycursor.execute(sql)
+    result = mycursor.fetchone()
+    if result:
+        sql = f"Delete from tblOwner WHERE userId={id}"
+        mycursor.execute(sql)
+        mydb.commit
```
Comment (AaronAfewerqi) [inline amplify/backend/function/OwnerAndRenterFunction/src/index.py:299]: parentheses is missing in mydb.commit

File: amplify/backend/function/OwnerAndRenterFunction/src/index.py [hunk 2]
```diff
@@ -229,57 +228,103 @@ def save_user(request_body,userPath):
...
+        }
+        status_code = 204
+  elif userPath == '/renter':
+    sql = f"select * from tblRenter where renterId={id}"
+    mycursor.execute(sql)
+    result = mycursor.fetchone()
+    if result:
+        sql = f"Delete from tblRenter WHERE renterId={id}"
+        mycursor.execute(sql)
+        mydb.commit
```
Comment (AaronAfewerqi) [inline amplify/backend/function/OwnerAndRenterFunction/src/index.py:317]: parentheses is missing in mydb.commit

**Review by AaronAfewerqi — CHANGES_REQUESTED** [review]
Efi, I think you only have to correct these two db.commit lines in the code. If db.commit becomes db.commit(), I think everything is okay and testing is the only missing part

File: amplify/backend/function/OwnerAndRenterFunction/src/index.py (hunk 1 above)
Comment (Efrr2022) [reply amplify/backend/function/OwnerAndRenterFunction/src/index.py:299]: You are right Arie.

**Comment by Efrr2022** [comment]
Pull request canceled without merging to adjust the errors.

**Pull request closed without merging**